app/src/components/v-table/table-header.vue

```diff
@@ -180,7 +180,8 @@ function getClassesForHeader(header: Header) {
 }
 
 function getTooltipForSortIcon(header: Header) {
-	return props.sort.by === header.value && props.sort.desc === false ? 'sort_desc' : 'sort_asc';
+	if (props.sort.by === null) return 'sort_asc';
+	return props.sort.by === header.value && props.sort.desc === false ? 'sort_desc' : 'disable_sort';
```

Comment:
Should be something like this as otherwise the unsorted header shows "Disable Sort" instead of "Sort Ascending":
```suggestion
	if (props.sort.by === null || props.sort.by !== header.value) return 'sort_asc';
	return props.sort.desc === false ? 'sort_desc' : 'disable_sort';
```